techniques/aria/ARIA11.html

```diff
@@ -22,13 +22,13 @@ <h1>Using ARIA landmarks to identify regions of a page</h1>
       <ul>
-         <li><code class="language-html">banner</code>: A region that contains mostly site-oriented content, such as the logo or a site-specific search tool.</li>
-         <li><code class="language-html">navigation</code>: A region that contains navigation links to other pages or different parts of the same page.</li>
-         <li><code class="language-html">main</code>: A region that contains a page's main content.</li>
-         <li><code class="language-html">region</code>: A region that contains a perceivable section of the page containing content that is sufficiently important for users to be able to navigate to the section. A <code class="language-html">region</code> landmark isn't exposed as a landmark region unless it has an accessible name, often provided using <code class="language-html">aria-label</code> or <code class="language-html">aria-labelledby</code>.</li>
-         <li><code class="language-html">form</code>: A region of the document that represents a collection of form-associated elements, some of which can represent editable values that can be submitted to a server for processing. A <code class="language-html">form</code> landmark isn't exposed as a landmark region unless it has an accessible name.</li></li>
-         <li><code class="language-html">search</code>: A region of the page containing search functionality.</li>
-         <li><code class="language-html">complementary</code>: Any section of the document that supports the main content, yet is separate and meaningful on its own.</li>
+         <li><code class="language-html">banner</code>: A region that contains mostly site-oriented content, rather than page-specific content.</li>
+         <li><code class="language-html">navigation</code>: A region that contains a collection of navigational elements (usually links) for navigating the document or related documents.</li>
+         <li><code class="language-html">main</code>: A region that contains the main content of a document.</li>
+         <li><code class="language-html">region</code>: A region that contains a perceivable section of the page containing content that is relevant to a specific, author-specified purpose and sufficiently important that users will likely want to be able to navigate to the section easily A <code class="language-html">region</code> landmark isn't exposed as a landmark region unless it has an accessible name.</li>
+         <li><code class="language-html">form</code>: A region that represents a collection of form-associated elements, some of which can represent editable values that can be submitted to a server for processing.</li>
+         <li><code class="language-html">search</code>: A region that contains a collection of items and objects that, as a whole, combine to create a search facility.</li>
```

Comment:
```suggestion
         <li><code class="language-html">search</code>: A region that contains a collection of items and objects that, as a whole, combine to create search functionality.</li>
```

Super editorial, but “facility” felt a little unnatural to me here? How about “functionality”, to match [APG’s landmark documentation](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/#:~:text=A%20search%20landmark%20contains%20a%20collection%20of%20items%20and%20objects%20that%2C%20as%20a%20whole%2C%20combine%20to%20create%20search%20functionality.)?